setup.sh

```diff
@@ -17,32 +17,72 @@
 # Enable "exit immediately if any command fails" option
```

Comment:
Can you put a comment at the top describing how to use the module? Looks great otherwise!

Reply:
Done.

I will add a README with more details for internal developer next.